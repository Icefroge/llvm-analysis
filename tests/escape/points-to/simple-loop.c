/*
  This test involves writes to a global.  These edges should not be
  killed when new writes are recorded.
 */
extern int a;
int ** pp;

void f() {
  int z;
  int g;
  int * p;
  p = &g;
  while(a) {
    p = &z;
    --a;
  }
}
