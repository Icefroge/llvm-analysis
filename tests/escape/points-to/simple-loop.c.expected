fromList [ ("%p", fromList [ "%z", "%g" ]) ]
